tests/applications_test.py

```diff
@@ -16,12 +17,38 @@
 from keras.applications import vgg16
 from keras.applications import vgg19
 from keras.applications import xception
+from keras_applications import resnet
```

Comment:
Should we simply apply this new testing style to all applications? It would be easier to have all or nothing.

Reply:
@fchollet, The testing style has been revised.